syft/serde.py

```diff
@@ -457,7 +461,7 @@ def _simplify_slice(my_slice: slice) -> Tuple[int, int, int]:
     Returns:
         list: a list holding the start, stop and step values
```

Comment:
It is not letting me review on the actual line but the arg for this function in the docs should be my_slice (slice). This is a problem throughout

Reply:
fixed